theseus_gui/src/pages/instance/Logs.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <Card class="log-card">
+    <div class="button-row">
+      <div>
+        <Button>
+          logs/latest.log
+        </Button>
```

Comment:
This is a dropdown right? should probably be moved to that (for multiple types of log files)